### PR TITLE
CBL-5997: Logging correlation ID

### PR DESCRIPTION
**Pull request by callumbirks**
This is an attempt at adding the logging of correlation ID, with as minimal changes as possible.
See #2105 for an example of the opposite.

### CODE REVIEW

File: Networking/BLIP/BLIPConnection.cc [hunk 1]
```diff
@@ -187,8 +187,7 @@ namespace litecore { namespace blip {
         // websocket::Delegate interface:
         virtual void onWebSocketConnect() override {
             _timeOpen.reset();
-            _connection->connected();
-            onWebSocketWriteable();
+            enqueue(FUNCTION_TO_QUEUE(BLIPIO::_onWebSocketConnect));
```
Comment (jianminzhao) [inline Networking/BLIP/BLIPConnection.cc:190]: This is not related to the title of the CBL ticket. Perhaps add a note in the commit comment.

**Review by jianminzhao — APPROVED** [review]

File: Networking/BLIP/BLIPConnection.cc (hunk 1 above)
Comment (callumbirks) [reply Networking/BLIP/BLIPConnection.cc:190]: Thanks, will do
